<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="2" ignored="0" total="2" passed="0">
  <reporter-output>
  </reporter-output>
  <suite name="Default suite" duration-ms="25885" started-at="2020-11-11T01:47:33Z" finished-at="2020-11-11T01:47:59Z">
    <groups>
      <group name="regression">
        <method signature="LoginTest.checkErrorMessage()[pri:0, instance:com.framework.testcase.LoginTest@10bdf5e5]" name="checkErrorMessage" class="com.framework.testcase.LoginTest"/>
      </group> <!-- regression -->
      <group name="smoke">
        <method signature="LoginTest.validLoginTest()[pri:0, instance:com.framework.testcase.LoginTest@10bdf5e5]" name="validLoginTest" class="com.framework.testcase.LoginTest"/>
      </group> <!-- smoke -->
    </groups>
    <test name="Default test" duration-ms="25885" started-at="2020-11-11T01:47:33Z" finished-at="2020-11-11T01:47:59Z">
      <class name="com.framework.testcase.LoginTest">
        <test-method status="PASS" signature="beforeSuite()[pri:0, instance:com.framework.testcase.LoginTest@10bdf5e5]" name="beforeSuite" is-config="true" duration-ms="171" started-at="2020-11-11T01:47:33Z" finished-at="2020-11-11T01:47:33Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeSuite -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.framework.testcase.LoginTest@10bdf5e5]" name="beforeMethod" is-config="true" duration-ms="2590" started-at="2020-11-11T01:47:33Z" finished-at="2020-11-11T01:47:36Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="checkErrorMessage()[pri:0, instance:com.framework.testcase.LoginTest@10bdf5e5]" name="checkErrorMessage" duration-ms="10977" started-at="2020-11-11T01:47:36Z" finished-at="2020-11-11T01:47:47Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at com.framework.seleniumBase.SeleniumBase.stepReport(SeleniumBase.java:148)
at com.framework.seleniumBase.SeleniumBase.click(SeleniumBase.java:60)
at com.framework.pages.LoginPage.loginWithInvalidCredentials(LoginPage.java:49)
at com.framework.testcase.LoginTest.checkErrorMessage(LoginTest.java:29)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- checkErrorMessage -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.framework.testcase.LoginTest@10bdf5e5]" name="afterMethod" is-config="true" duration-ms="57" started-at="2020-11-11T01:47:47Z" finished-at="2020-11-11T01:47:47Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="beforeMethod()[pri:0, instance:com.framework.testcase.LoginTest@10bdf5e5]" name="beforeMethod" is-config="true" duration-ms="1335" started-at="2020-11-11T01:47:47Z" finished-at="2020-11-11T01:47:48Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- beforeMethod -->
        <test-method status="FAIL" signature="validLoginTest()[pri:0, instance:com.framework.testcase.LoginTest@10bdf5e5]" name="validLoginTest" duration-ms="10857" started-at="2020-11-11T01:47:48Z" finished-at="2020-11-11T01:47:59Z">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[null]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: null
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.fail(Assert.java:103)
at com.framework.seleniumBase.SeleniumBase.stepReport(SeleniumBase.java:148)
at com.framework.seleniumBase.SeleniumBase.click(SeleniumBase.java:60)
at com.framework.pages.LoginPage.loginWithValidCredentials(LoginPage.java:40)
at com.framework.testcase.LoginTest.validLoginTest(LoginTest.java:22)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:132)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:230)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:76)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validLoginTest -->
        <test-method status="PASS" signature="afterMethod()[pri:0, instance:com.framework.testcase.LoginTest@10bdf5e5]" name="afterMethod" is-config="true" duration-ms="54" started-at="2020-11-11T01:47:59Z" finished-at="2020-11-11T01:47:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterMethod -->
        <test-method status="PASS" signature="afterSuite()[pri:0, instance:com.framework.testcase.LoginTest@10bdf5e5]" name="afterSuite" is-config="true" duration-ms="271" started-at="2020-11-11T01:47:59Z" finished-at="2020-11-11T01:47:59Z">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- afterSuite -->
      </class> <!-- com.framework.testcase.LoginTest -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
